pyproject.toml

```diff
@@ -193,7 +193,7 @@ mkdocs-include-markdown-plugin = ">=4.0.4"
 # Use Jinja2 templating in docs - see optional-settings.md and required-settings.md
 mkdocs-macros-plugin = "~1.0.5"
 # Material for mkdocs theme
-mkdocs-material = "~9.5.35"
+mkdocs-material = "~9.5.39"
```

Comment:
Should also update `docs/requirements.txt` and `example_app/docs/requirements.txt` accordingly. Looks like we missed that in the last update.